src/main/java/org/terasology/launcher/gui/javafx/GameReleaseCell.java

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.launcher.gui.javafx;
+
+import javafx.scene.control.ListCell;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import org.terasology.launcher.model.Build;
+import org.terasology.launcher.model.GameIdentifier;
+import org.terasology.launcher.model.GameRelease;
+import org.terasology.launcher.util.BundleUtils;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Set;
+
+/**
+ * Custom {@link ListCell} used to display a {@link GameRelease} along with its installation status.
+ */
+final class GameReleaseCell extends ListCell<GameRelease> {
+    private static final Image ICON_CHECK = BundleUtils.getFxImage("icon_check");
+    private static final DateFormat DATE_FORMAT = new SimpleDateFormat("yyyyMMdd");
```

Comment:
Out of curiosity, why is this yyyyMMdd instead of yyyymmdd?

Reply:
because `mm` stands for minutes :shrug: see https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html